From a3267be192e6b47131a5c08376ad13d3356d9109 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Mon, 19 Aug 2019 21:13:01 +0200
Subject: [PATCH 058/152] network: Don't omit parens when constructing

While it is legal to omit parentheses when invoking a constructor
with no arguments, we generally avoid that in our coding style.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/710
---
 js/ui/status/network.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/status/network.js b/js/ui/status/network.js
index 7cff22c6e..4bb012d88 100644
--- a/js/ui/status/network.js
+++ b/js/ui/status/network.js
@@ -874,7 +874,7 @@ class NMWirelessDialog extends ModalDialog.ModalDialog {
         this._airplaneHeadline = new St.Label({ style_class: 'nm-dialog-airplane-headline headline' });
         this._airplaneText = new St.Label({ style_class: 'nm-dialog-airplane-text' });
 
-        let airplaneSubStack = new St.Widget({ layout_manager: new Clutter.BinLayout });
+        let airplaneSubStack = new St.Widget({ layout_manager: new Clutter.BinLayout() });
         this._airplaneButton = new St.Button({ style_class: 'modal-dialog-button button' });
         this._airplaneButton.connect('clicked', () => {
             if (this._rfkill.airplaneMode)
-- 
2.23.0

